client/ayon_colorbleed/__init__.py

```diff
@@ -3,9 +3,12 @@
 from ayon_core.addon import AYONAddon
 from ayon_core.addon.interfaces import IPluginPaths
 
+from .version import __version__
 
 class ColorbleedAddon(AYONAddon, IPluginPaths):
     name = "colorbleed"
+    title = "Colorbleed"
+    version = __version__
```

Comment:
Am I going crazy - or why does this show as diff? :) The [same line](https://github.com/BigRoy/ayon-colorbleed/blob/main/client/ayon_colorbleed/__init__.py#L11) is already there?